app/assets/javascripts/sidebar.js

```diff
@@ -1,14 +1,5 @@
-function toggle_sidebar() { 
-  if ($('#sidebar').hasClass('d-none')) { 
-    $('#sidebar').removeClass('d-none') 
-                 .removeClass('d-sm-block') 
-                 .removeClass('d-md-none') 
-                 .removeClass('d-lg-block') 
-  } else { 
-    $('#sidebar').addClass('d-none') 
-                 .addClass('d-sm-block') 
-                 .addClass('d-lg-block') 
-                 .addClass('d-md-none') 
-  } 
+function toggle_sidebar() {
+  $(".accordion button").attr("aria-expanded", function (index, attr) {
```

Comment:
This is really nice and so much more compact. Are we sure, though, that there are no other templates in the codebase where we were toggling sidebars? Other places we may have to update too, so that the new code works there too? Thank you!

Reply:
It seems that this is the only template for toggling sidebars, I'm not sure though. @jywarren 
Apologies for delay.

Reply:
This is the only template using this code. We're good!

Reply:
Okay, thanks for the clarification @TildaDares.